src/boot/efi/secure-boot.c

```diff
@@ -12,6 +12,15 @@ BOOLEAN secure_boot_enabled(void) {
         return !EFI_ERROR(err) && secure;
 }
 
+BOOLEAN setup_mode_enabled(void) {
```

Comment:
i'd split this funciton out into a separate commit, which we can merge early on. this is a good change either way, and should be uncontroversial to merge